pallets/asset/src/lib.rs

```diff
@@ -1904,7 +1908,7 @@ impl<T: Config> Module<T> {
         portfolio_kind: PortfolioKind,
     ) -> DispatchResult {
         // Ensure origin is agent with custody and permissions for default portfolio.
```

Comment:
The comment here needs to be updated, since it might not be the default portfolio.